idea_town/frontend/static-src/app/templates/landing-page.js

```diff
@@ -0,0 +1,28 @@
+export default `
+  <div>
+  {{^loggedIn}}
+    <div class="firefox-logo"></div>
+    <h1 class="hero">Introducing Idea Town</h1>
+    <h2 class="sub-hero">We're building the next generation of<br> Firefox features and we want your feedback! <br>Get started with a Firefox Account.</h2>
+    <div class="cta-layout-wrapper">
+      <div class="cta-layout">
+        <a href="/accounts/login/?next=/home"><button class="button large primary">Sign up</button></a>
+        <a href="/accounts/login/?next=/home" class="fxa-alternate">Already have an account? Sign in.</a>
+        <p class="cta-legal">By proceeding, you agree to the Terms of Service and Privacy Notice of Idea Town.</p>
```

Comment:
Should "Terms of Service" and "Privacy Notice" be linked here (even if just to "#" for now).